test/vanilla/ignite-generate.test.ts

```diff
@@ -57,37 +57,39 @@ describe("ignite-cli generate", () => {
         "
       `)
       expect(read(`${TEMP_DIR}/app/models/Pizza.ts`)).toMatchInlineSnapshot(`
-        "import { Instance, SnapshotIn, SnapshotOut, types } from \\"mobx-state-tree\\"
-        import { withSetPropAction } from \\"./helpers/withSetPropAction\\"
+"import { Instance, SnapshotIn, SnapshotOut, types } from \\"mobx-state-tree\\"
+import { withSetPropAction } from \\"./helpers/withSetPropAction\\"
 
```

Comment:
I would guess this diff came from disabling Prettier in Jest, so maybe we should find a different way to fix that, or else it will cause a lot more noisy churn after the fact with other inline snapshots.

Reply:
Another approach would be ec682f836daea6f9db7009d0205ecfe6cd3f8e48, which [uses the workaround of a separate Prettier version for Jest](https://github.com/jestjs/jest/issues/14305#issuecomment-1668738936). I set `prettier-2` to be `npm:prettier@2.8.8` which I saw was the last version before the update in https://github.com/infinitered/ignite/pull/2761/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519. 

Didn't seem to change the whitespace diff here, so maybe there's no real issue. Figured I'd at least provide the commit and we could easily drop it if the `prettierPath: null` is acceptable.